games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit cmake-utils eutils gnome2-utils user vcs-snapshot
```

Comment:
I am a bit irritated is it about changing the order of the eclasses in the inherit line or about removing the use of the games eclass?